aptos-move/framework/aptos-framework/sources/jwks.move

```diff
@@ -184,20 +186,70 @@ module aptos_framework::jwks {
         old_config_url
     }
 
-    /// Remove an OIDC provider from the `SupportedOIDCProviders` resource.
-    /// Can only be called in a governance proposal.
-    /// Returns the old config URL of the provider, if any, as an `Option`.
+    /// Used in on-chain governances to update the supported OIDC providers, effective starting next epoch.
+    /// Example usage:
+    /// ```
+    /// aptos_framework::jwks::upsert_oidc_provider_for_next_epoch(
+    ///     &framework_signer,
+    ///     b"https://accounts.google.com",
+    ///     b"https://accounts.google.com/.well-known/openid-configuration"
+    /// );
+    /// aptos_framework::aptos_governance::reconfigure(&framework_signer);
+    /// ```
+    public fun upsert_oidc_provider_for_next_epoch(fx: &signer, name: vector<u8>, config_url: vector<u8>): Option<vector<u8>> acquires SupportedOIDCProviders {
```

Comment:
Does upserting or deleting oidc trigger epoch change?

Reply:
i think it should